src/provider/model/Environment.ts

```diff
@@ -3,7 +3,7 @@ import {Signal} from 'openfin-service-signal';
 
 import {Application} from '../../client/main';
 
-import {AppWindow} from './AppWindow';
+import {AppConnection} from './AppConnection';
 import {ContextChannel} from './ContextChannel';
 
 export enum EntityType {
```

Comment:
Should we be using the term "Entity" at all? Couldn't we use "Connection" everywhere we're using "Entity"?

Reply:
When you've got `registerConnection` commented with " Registers an entity in the model" it feels kinda muddled. Enity everywhere might be better, then no confusion with FDC3 connections, and it's in line with runtime terminology